src/journal/src/Journal/Internal/Utils.hs

```diff
@@ -21,13 +21,11 @@ import System.Posix.IO
        )
 import System.Posix.Types (Fd)
 
+import Assert
```

Comment:
Export `module Assert` in `Utils` module, so we don't have to import `Assert` module everywhere?